checking_account.rb

```diff
@@ -0,0 +1,33 @@
+module Bank
+  class CheckingAccount < Account
+    attr_reader :used_checks, :balance, :account_id, :owner_id
```

Comment:
I don't think you need all of these readers in this class since some of them are already in the `Account` class